Cargo.toml

```diff
@@ -72,7 +72,7 @@ nix = "0.26.1"
 openssl = "0.10.49"
 once_cell = "1.16.0"
 os-release = "0.1.0"
-ostree-ext = "0.10.0"
+ostree-ext = { git = "https://github.com/ostreedev/ostree-rs-ext.git"}
```

Comment:
OK this can be updated to use the now-just-published version 
`ostree-ext = "0.11.0"`